packages/Switches/src/index.tsx

```diff
@@ -0,0 +1,96 @@
+import withUtils from "@blaze-react/utils";
+import React, { FunctionComponent, useState } from "react";
+
+interface ISwitchesProps {
+  labelPosition?: string;
+  options?: any[] | object;
+  modifier?: string;
+  returnBoolean?: boolean;
+  onChange: (...args: any) => void;
+  utils: {
+    uniqueId: (element: any) => string;
+  };
+}
+const Switches: FunctionComponent<ISwitchesProps> = ({
+  labelPosition,
+  onChange,
+  options,
+  modifier,
+  returnBoolean,
+  utils: { uniqueId },
+  ...attrs
+}): JSX.Element => {
+  const formatedOptions = Array.isArray(options) ? options : [options];
+
+  const [data, setData]: any = useState(formatedOptions);
+
+  const toggle = ({
+    event,
+    item,
+    key
+  }: {
+    event: React.ChangeEvent<HTMLInputElement>;
+    item: object | any;
+    key: number;
+  }): void => {
+    if (item.disabled) {
+      return;
+    }
+
+    data[key].checked = !item.checked;
+    setData([...data]);
+
+    let checked = data.filter((option: any): boolean => option.checked);
+
+    if (returnBoolean) {
+      checked = !!checked.length;
+    }
+
```

Comment:
If looking for a single value can use a .find, also the checked value passed to html is expected to be boolean, dont understand why it sometimes is or isnt boolean within this function. otherwise approved.

Reply:
The logic of the switches returns a Boolean value if it is a single switch, or an array of active switches if it is a group of options, that is why I use  filter